default/etc/e-smith/templates/etc/httpd/conf.d/nethserver.conf/10tls_policy_20180330

```diff
@@ -4,9 +4,12 @@
  if ($tlsPolicy >= 20180330) {
      $OUT .= qq(
```

Comment:
As the string does not need vars interpolation, I suggest to disable it and use just `q()` (or single quotes) so we do not need double escape for backslash `\`